JamGame/Assets/Scripts/Overlay/OverlayManager.cs

```diff
@@ -0,0 +1,41 @@
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public interface IOverlayManager
+{
+    public void RevertAllOverlays();
+
+    public void ApplyOverlay<O>(O overlay) where O : class, IOverlay;
+}
+
+public class OverlayManager : MonoBehaviour, IOverlayManager
+{
+    // TODO: Update in runtime.
+    private List<IOverlayRenderer> overlayRenderers;
+
+    private void Start()
+    {
+        overlayRenderers = GetComponentsInChildren<IOverlayRenderer>().ToList();
```

Comment:
For sure, it do not will get components in disable objects or if component disables

Reply:
When Employees will spawn in location they will notify OverlayManager about this

Reply:
The current approach is for testing purposes mainly